docs/user/glossary.rst

```diff
@@ -82,13 +77,9 @@ Terminology
 
 .. versionadded:: 4.5
 
-Flagging certain glossary terms as terminology puts them in
-all glossary languages. Use this to flag important terms which should be
-translated consistently.
-
-The terms can be flagged in the source language using :guilabel:`Tools` ↓
-:guilabel:`Mark as terminology`. In the background this toggles the
-``terminology`` flag of the string.
+Flagging certain glossary terms as ``terminology`` manually, or by
+using :guilabel:`Tools` ↓:guilabel:`Mark as terminology` puts them in all
+glossary languages. Use this for terms that should be translated consistently.
```

Comment:
I'd go with something like this:

```suggestion
using :guilabel:`Tools` ↓:guilabel:`Mark as terminology` adds them to all 
the languages in the glossary. Use this for important terms which translation
to each and every language should be well thought and should have consistent meaning
across all of them .
```

<hr>

Also this one completely misses that this option is available only to privileged users.

Reply:
I think I changed this afterwards. Will have to check.

Reply:
The second part is fine, but the section still lacks a note that it can be used only by privileged users by default (probably in a different paragraph).
And the same thing as for bulk-edit etc as above...

Reply:
closing as outdated